src/dist/manifestation.rs

```diff
@@ -11,22 +11,19 @@ use crate::dist::component::{Components, Package, TarGzPackage, TarXzPackage, Tr
 use crate::dist::config::Config;
 use crate::dist::dist::{Profile, TargetTriple, DEFAULT_DIST_SERVER};
 use crate::dist::download::{DownloadCfg, File};
-use crate::dist::manifest::{Component, Manifest, TargetedPackage};
+use crate::dist::manifest::{Component, CompressionKind, Manifest, TargetedPackage};
 use crate::dist::notifications::*;
 use crate::dist::prefix::InstallPrefix;
 use crate::dist::temp;
 use crate::errors::*;
 use crate::process;
 use crate::utils::utils;
 
+use super::component::TarZStdPackage;
```

Comment:
not clear why this and only this package type appears here - shouldn't it be on line 10 with the others?

Reply:
Looks like when rust-analyzer auto-added the import it didn't group it for some reason.  Fixed.